fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -292,6 +293,10 @@ public class Loan extends AbstractPersistableCustom {
     @Column(name = "loan_product_counter")
     private Integer loanProductCounter;
 
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "delinquency_range_id", nullable = true)
+    private DelinquencyRange delinquencyRange;
```

Comment:
Maybe I misunderstood but isn't the LoanDelinquencyTags storing which delinquency range a loan is related to? What's the point of this? Thanks.

Reply:
It is to represent the current tag, considering LoanDelinquencyTags could work for the next scenarios:
1. Loan without delinquency tag because It is active (no delinquency yet)
2. Loan with one or more delinquency tags (history) and you can take the latest one to know the current delinquency tag,
But In the scenario, after go out of delinquency, when the loan goes back to active, how you can reflect that? No delinquency

Plus It can help to know quickly If the loan has or not a tag, If It has a tag, exactly which is it.